src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx

```diff
@@ -263,7 +263,10 @@ export default function TechnicalIntegration() {
 
   const csvPreview = (files: File[]) => {
     files
-      .filter((file: File) => file.type === 'text/csv')
+      .filter(
+        (file: File) =>
+          file.type === 'text/csv' || file.type === 'application/vnd.ms-excel'
+      )
```

Comment:
HI @kunalgaurav-bmw aren't you here just enabling an additional format (excel) for the upload?
The original issue https://github.com/eclipse-tractusx/portal-frontend/issues/961 states that the role upload with the already enabled format, csv, doesn't work. Enabling an additional format won't solve that issue.
Can you even reproduce the issue in Firefox?
I just tired with the version 115.3.1esr and I can't reproduce the issue, the role upload works.

Reply:
Hi @evegufy Yes, I tested the issue on version 129.0 and I was able to reproduce the problem where .csv file (I tried with the "template_app_role_upload.csv" downloaded from the same role upload page) could be successfully drag-and-dropped and role upload in chrome but were not working in Firefox. This indicates a compatibility issue specific to Firefox or might have stricter file validation rule  .
The reason I added the "application/vnd.ms-excel" MIME type to support a wider range of formats and to enhance the compatibility. And with this changes it working fine for Firefox same as chrome.